app/styles/nypr-member-center.scss

```diff
@@ -13,6 +13,8 @@
 @import "nypr-member-center/current-status-detail";
 @import "nypr-member-center/loading-animation";
 
+@import "nypr-account-modal";
+
```

Comment:
I don't think you want to import this here, since it will double up the produced CSS.